src/components/Tooltip/index.js

```diff
@@ -44,6 +44,21 @@ class Tooltip extends Component {
     };
   }
 
+  /*
+   * If any specific value is passed via `direction` props, than that value should be used
+   * If `auto` value is passed via `direction` prop, then calculated direction value will be used
+   */
+  getDirection = () => {
+    const { direction } = this.props;
+    const { actualDirection } = this.state;
```

Comment:
I just noticed that we have both `this.state.actualDirection` and `this.actualDirection`... weird.

I don't see us updating the `actualDirection` state either.

Could you please refactor to use `this.actualDirection` and deprecate `this.state.actualDirection`?

Reply:
@mlampedx I have got rid of `this.actualDirection` instead of variable in `state`, as it more suitable to manipulate component via state whenever necessary

Reply:
Excellent, thank you!